### PR TITLE
refac: rename webapi test projects + update namespaces

### PR DESCRIPTION
**Pull request by lasrinnil**
## Description

FROM
image

TO
image

## References

* #0000

### CODE REVIEW

File: source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj [hunk 1]
```diff
@@ -31,7 +31,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\IntegrationTests\IntegrationTests.csproj" />
+      <ProjectReference Include="..\WebApi.IntegrationTests\WebApi.IntegrationTests.csproj" />
```
Comment (BjarkeMeier) [inline source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj:34]: Why do we have this ref?

File: .github/actions/dotnet-tests-prepare-outputs/action.yml [hunk 2]
```diff
@@ -33,23 +33,23 @@ runs:
       shell: bash
       run: |
         dotnet publish \
-          '.\source\dotnet\Services\IntegrationTests\IntegrationTests.csproj' \
+          '.\source\dotnet\Services\WebApi.IntegrationTests\WebApi.IntegrationTests.csproj' \
```
Comment (BjarkeMeier) [inline .github/actions/dotnet-tests-prepare-outputs/action.yml:36]: What about integration and unit tests of process manager?

File: source/dotnet/Services/WebApi.IntegrationTests/WebApi.IntegrationTests.csproj [hunk 3]
```diff
@@ -26,8 +26,8 @@ limitations under the License.
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <AssemblyName>Energinet.DataHub.Wholesale.IntegrationTests</AssemblyName>
-    <RootNamespace>Energinet.DataHub.Wholesale.IntegrationTests</RootNamespace>
+    <AssemblyName>Energinet.DataHub.Wholesale.WebApi.IntegrationTests</AssemblyName>
+    <RootNamespace>Energinet.DataHub.Wholesale.WebApi.IntegrationTests</RootNamespace>
```
Comment (BjarkeMeier) [inline source/dotnet/Services/WebApi.IntegrationTests/WebApi.IntegrationTests.csproj:30]: Namespace updates in an upcoming PR?

**Review by BjarkeMeier — APPROVED** [review]

File: source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+-->
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+  To ensure code coverage tooling is available on build agents we have to use publish in workflow.
+  This can cause an error which we ignore using the follow setting.
+  See https://stackoverflow.com/questions/69919664/publish-error-found-multiple-publish-output-files-with-the-same-relative-path/69919694#69919694
+  -->
+  <PropertyGroup>
+    <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
+  </PropertyGroup>
```
Comment (BjarkeMeier) [inline source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj:24]: Just curious: Why (and what is it)?

File: source/dotnet/Services/WebApi.UnitTests/TestHelpers/EmbeddedResources.cs [hunk 5]
```diff
@@ -25,7 +25,8 @@ public static class EmbeddedResources
     /// </summary>
     public static Stream GetStream(string relativeDocumentPath)
     {
-        var resourceName = $"Energinet.DataHub.Wholesale.Tests.{relativeDocumentPath}";
+        const string assemblyRootNamespace = "Energinet.DataHub.Wholesale.WebApi.UnitTests";
```
Comment (BjarkeMeier) [inline source/dotnet/Services/WebApi.UnitTests/TestHelpers/EmbeddedResources.cs:28]: fyi: I like this, https://stackoverflow.com/a/33651494/8219438

**Review by johevemi — APPROVED** [review]

File: source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj (hunk 1 above)
Comment (lasrinnil) [reply source/dotnet/Services/ProcessManager.IntegrationTests/ProcessManager.IntegrationTests.csproj:51]: Some things are still shared and will not be addressed in this pr.

**Review by dstenroejl — APPROVED** [review]

**Pull request merged**